cmd/traefik/traefik.go

```diff
@@ -100,12 +100,11 @@ func runCmd(staticConfiguration *static.Configuration) error {
 
 	log.WithoutContext().Infof("Traefik version %s built on %s", version.Version, version.BuildDate)
 
-	jsonConf, err := json.Marshal(staticConfiguration)
+	redactedStaticConfiguration, err := redactor.RemoveCredentials(staticConfiguration)
 	if err != nil {
 		log.WithoutContext().Errorf("Could not marshal static configuration: %v", err)
```

Comment:
```suggestion
		log.WithoutContext().Errorf("Redacting static configuration: %v", err)
```

Reply:
I understand that we might want the term "redact" rather than "marshal" but don't you think we should keep the "Could not" form?

Reply:
@jspdown You are right! I'm adjusting the suggestion.